selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CheckOpenFileFeatureTest.java

```diff
@@ -85,7 +87,7 @@ public void checkOpenFileFeatureTest() throws Exception {
     String currentWin = seleniumWebDriver.getWindowHandle();
     seleniumWebDriver.switchToNoneCurrentWindow(currentWin);
```

Comment:
We can move `SeleniumWebDriver#switchToNoneCurrentWindow(currentWin)` method to the **SeleniumWebDriverHelper** as well, can't we?